src/test/java/org/opensearch/neuralsearch/sparse/SparseSearchingIT.java

```diff
@@ -670,6 +670,36 @@ public void testSearchWithCustomizedQuantizationCeil() {
         verifyTopDocScoreExpected(TEST_INDEX_NAME, neuralSparseQueryBuilder9, 30.0f);
     }
 
+    public void testSearchDocumentsWithNoMethodParameters() throws Exception {
```

Comment:
sorry, I didn't catch this before, do you think we need to verify multi-node scenario as the deserialization is touched.

Reply:
Thank you, let's wait for CI to pass